pkg/types/types.go

```diff
@@ -193,6 +193,7 @@ type ResourcePool interface {
 	GetMounts(deviceIDs []string) []*pluginapi.Mount
 	StoreDeviceInfoFile(resourceNamePrefix string) error
 	CleanDeviceInfoFile(resourceNamePrefix string) error
+	GetPoolKind() string
```

Comment:
GetPoolType ? or Better yet, GetPoolDeviceType ?

Reply:
I prefer to keep it as is because it's 'kind' in CDI spec

Reply:
so GetPoolCDIKind ? 

Kind lacks context here IMO

Reply:
Let's simplify it to `GetPoolCDIKind`

Reply:
> Let's simplify it to `GetPoolCDIKind`

I mean `GetCDIKind`